### ISSUE TITLE
Example formatting is intimidating

### ISSUE THREAD

**Issue by timbertson-zd** [arianvp/dhall-kubernetes#87]
Are the examples (e.g. https://github.com/dhall-lang/dhall-kubernetes/blob/master/examples/deploymentSimple.dhall ) auto-formatted somehow? These were very offputting to me as the first example I was exposed to, the indentation makes it really hard to parse and implies much more complexity (nesting) than there actually appears to be.

I manually reformatted as https://gist.github.com/timbertson-zd/accefe1f69fd28c77238e43bc80ac3b2. To my (unfamiliar) eyes it looks much more approachable.

**Comment by ari-becker** [comment]
> 
> 
> Are the examples (e.g. https://github.com/dhall-lang/dhall-kubernetes/blob/master/examples/deploymentSimple.dhall ) auto-formatted somehow? These were very offputting to me as the first example I was exposed to, the indentation makes it really hard to parse and implies much more complexity (nesting) than there actually appears to be.
> 
> I manually reformatted as https://gist.github.com/timbertson-zd/accefe1f69fd28c77238e43bc80ac3b2. To my (unfamiliar) eyes it looks much more approachable.

The Dhall (Haskell) executable has a formatter which establishes a standard formatting for the language (similar to `go fmt` or `terraform fmt`). People don't have to use it, but it helps for shared work, as all auto-formatters do. Hand-formatting will, strictly speaking, almost always get you a nicer result, at the expense of occasional mistakes, trying to persuade other people you work with to take the time to format nicely, etc.

For what it's worth, I also found the standard formatting jarring when I first started writing Dhall - I got used to it over time, and now the standardization makes standard-formatted code easier to read for me.

**Comment by Gabriella439** [comment]
I think it's important to note here that once `dhall-kubernetes` merges this:

https://github.com/dhall-lang/dhall-kubernetes/pull/84

... the examples get nicer by using the record completion operator.  See this for example:

https://github.com/dhall-lang/dhall-kubernetes/blob/4ef46543ceff0e10978cca542952c56ee6e47b93/examples/aws-iam-authenticator-chart.dhall

**Comment by craig-day** [comment]
yeah... the schemas will look so much nicer once we can do record completion. So much so, that I'm actually referencing the commit from #84 in my files and it killed every usage of merging `//` and cleaned up the schemas a ton.

**Comment by timbertson-zd** [comment]
Thanks @Gabriel439, sounds like that'll improve things dramatically
